docs/guides/observability/tracing.adoc

```diff
@@ -162,8 +162,37 @@ For more information, see the https://www.w3.org/TR/trace-context/#security-cons
 == Tracing in Kubernetes environment
 When the tracing is enabled when using the {project_name} Operator, certain information about the deployment is propagated to the underlying containers.
 
-NOTE: There is no support for tracing configuration in {project_name} CR yet, so the `additionalOptions` can be used to the `tracing-enabled` property and other tracing options.
+=== Configuration via Keycloak CR
```

Comment:
We're usually trying to keep any Operator related docs in the Operator guide. Since it's just a simple paragraph, IMHO we could leverage the [advance config guide](https://github.com/keycloak/keycloak/blob/main/docs/guides/operator/advanced-configuration.adoc) for it (we might want to split it in the future into several guides).

Reply:
I can include it in there in some simpler version. But I'd like to have it even present in more advanced form in the tracing guide. @vmuzikar does it work for you?